data/tutorials/platform/bp_03_run_executables_and_tests.md

```diff
@@ -39,6 +39,14 @@ For instance, if you've put your `dune` file in `bin/dune` with the following co
 
 You can run it with `dune exec bin/main.exe` or `dune exec my-app`.
 
+## Building the Project When Files Changes
+
+The `dune exec <executable_path>.exe` could take some time to compile a project when multiple files are involved. It could be interesting to have a process which recompiles each file as soon as they are changed. Then after the last saved file, it is possible that it remains only a last file to compile or just the linking process of all files and libraries.
```

Comment:
```suggestion
The `dune exec <executable_path>.exe` could take some time to compile a project when multiple files are involved. It could be interesting to have a process that recompiles each file as soon as they are changed. Then after the last saved file, it is possible that it remains only a last file to compile or just the linking process of all files and libraries.
```
In the last part of this sentence, do you mean:

It is possible that only one file remains to be compiled

Also, I don't understand what you mean by "just the linking process of all..."

Do you mean that perhaps only one file and the linking process remain? 

Please clarify and we'll work on this sentence together. 😊